test/reputation-system/reputation-mining-client/client-auto-functionality.js

```diff
@@ -80,7 +80,8 @@ process.env.SOLIDITY_COVERAGE
           realProviderPort,
           minerAddress: MINER1,
           useJsTree: true,
-          auto: true
+          auto: true,
+          oracle: false
```

Comment:
Maybe I missed it but what's the purpose of this flag? Seems like it's the same as setting `auto = false`, but maybe not? Does it tell the miner to run the express server? I guess you'd want to be able to auto-mine without exposing the server so it does make sense to have them be orthogonal features.

Reply:
Yeah, that's exactly what we're doing, and is probably ultimately what we should recommend people do - no server exposed means no private keys exposed if there's an express exploit.